lib/scarpe/stack.rb

```diff
@@ -12,7 +12,7 @@ class Stack < Scarpe::Widget
 
     display_properties :width, :height, :margin, :padding, :scroll, :margin_top, :options
 
-    def initialize(width: nil, height: nil, margin: nil, padding: nil, scroll: false, margin_top: nil, **options, &block)
+    def initialize(width: nil, height: "100%", margin: nil, padding: nil, scroll: false, margin_top: nil, **options, &block)
```

Comment:
is there a reason for changing height:100% here? this is breaking some examples like in our previous scenarios

```
  stack width: 10 do
    background "green"
  end
  stack width: 20 do
    background "red"
  end
  ```
  this used to work well and good but now purple is getting 100% height.. so defining height for every stack is becoming a necessity.

Reply:
nvm abt this, i missed tht discord msg